sdk/typescript/src/zklogin/jwt-utils.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Claim } from './types.js';
+
+export function base64UrlCharTo6Bits(base64UrlChar: string): number[] {
```

Comment:
the base64Url logic should probably be extracted to its own file

Reply:
I will keep this in the same file for now, just because it is used only here and not export it. If we need to use it elsewhere we can then move it to it's own file